guide/inheritance-wallet/overview.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: guide
+title: Overview
+description: An introduction to the use case, personas and scope of the inheritance wallet.
+nav_order: 1
+parent: Inheritance wallet
+permalink: /guide/inheritance-wallet/overview/
+main_classes: -no-top-padding
+
+---
+
+<!--
+
+Editor's notes
+
+This page covers ....  
+
+Illustration sources
+
+https://www.figma.com/file/h5GP5v5dYfpXXfEUXf6nvC/Family-inheritance-wallet?type=design&node-id=5542%3A2119&mode=design&t=sBtcvrDzb8MPtWaK-1
+
+-->
+
+# Overview 
+{: .no_toc }
+
+---
+
+<div class="glossary-toc" markdown="1">
+ * Table of contents
+{:toc}
+</div>
+
+---
+
+The target audience for the inheritance wallet are families who are familiar with bitcoin and hold the majority of their net worth in bitcoin. They want to take advantage of the security assurances of multi-key wallets and are comfortable with using more advanced technical features and human processes for inheritance planning.  
+
+For other users, such a setup might be too advanced. When choosing a specific setup, the decision should be based on factors such as the amount of bitcoin they hold, what their family situation looks like, where they live or how knowledgeable and comfortable they are with bitcoin in general. This line of thinking applies not only to the initial bitcoin holders themselves, but also to their heirs, as they will have to recover the funds at some point.
```

Comment:
We could link to the [personal finance page](https://bitcoin.design/guide/designing-products/personal-finance/), which provides a framework for choosing the right security setup for your use case.

Reminder to self to update the personal finance page at some point with a link to this inheritance wallet reference design.

Reply:
Also, the first sentence is very long. Considering our international audience with many for whom English is the second language, I think it's good to keep text short and direct. Up to you whether you want to adjust it or not.